spec/views/posts/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
```

Comment:
Hmmm. It looks like you got a auto-generated spec files here, not all of which you're really going to use. In a project like this, I would tend to lean towards lots of model tests, occasional controller tests if you want to test what's happening to the data, and using your integration tests to look at this kind of thing. Your testing structure should depend on the project, and there seems to me a lot in here that isn't going to provide much value.